db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -721,6 +743,58 @@ class NonBatchedOpsStressTest : public StressTest {
                   Slice(value).ToString(true /* hex */).c_str());
           is_consistent = false;
         }
+
+        if (is_consistent && use_txn) {
+          assert(!expected_values.empty());
+          if (!expected_values[i].has_value()) {
+            continue;
+          }
+          ExpectedValue expected = expected_values[i].value();
+          char expected_value[100];
+          if (s.ok()) {
+            Slice from_db_slice(values[i]);
+            if (!ExpectedValueHelper::MustHaveExisted(expected, expected)) {
```

Comment:
I am fine with using ExpectedValueHelper util on expected value. However, `!ExpectedValueHelper::MustHaveExisted` means "may have existed", which is not `ExpectedValueHelper::MustHaveNotExisted()`.   I will update my API comment better.

Reply:
There we go https://github.com/facebook/rocksdb/pull/11456

Reply:
Thank you for catching this and the PR.